WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -461,7 +461,7 @@ province_event = {
 				character_event = { id = zusk_event.36 } #TODO: missing zusk_event.36
```

Comment:
Could zusk_event.20 be the event? Would be cool if the problem was only wrong ID being called :relieved:.

Reply:
No, definitely not. That being said, I noticed zusk_event.35 should be added.

Reply:
Could you send me the original Slavic Migration files sent by Zusk (if you still have them of course)?

Reply:
I don't have the files sent by Zusk himself since he never sent any originals to me, and at the time there was no separate Slavic Migration file.

Reply:
I mean, the files which contain more of his work than there's in the repo, e.g. zusk_event.35 which you've just added.

Reply:
That file no longer exist, now that I added zusk_event.35  everything has been transferred, so the file will just be the carbon copy of what you already have (but not updated and standardised in terms of tabulation).

Reply:
Okay, good to know. Now that I've found the localisation entry for zusk_event.36, it's shouldn't be hard to write it :).

Reply:
If you look at the localisation for 20, 21, 35 and 36 you will see that they are similar.

Reply:
Maybe 36 had similar code? I have to fix 35 first anyway.

Was the "TODO: missing zusk_event.36" added before the event was restored to master?

Reply:
The localisation should give clues. zusk_event.35 was added for the first time to the master today, if that is what you refer to. When it comes to the TODO bit, I added it when introducing it.

Reply:
It seems strange to me that this event is missing, as it already has the localisation and other events are more or less finished.
Could you send me your local files with Slavic Migration code? I know what you said previously, but I just want to be sure it's missing and should be reconstructed basing on localisation.
The event could have been e.g. placed in a separate file for some reason.

Reply:
There is only one file, but yes, I will upload it.

Reply:
So yeah, it's all the same in the old file. I'm going to start reconstructing then.

Reply:
I warned you. ;)